### DISCUSSION TITLE
Console formatting

### DISCUSSION THREAD

**Discussion by jasonLaster** [RecordReplay/devtools#2371] [Ideas]
https://levelup.gitconnected.com/add-styles-and-formatting-to-your-console-log-messages-in-javascript-5f14819b1c5d

**Comment by jonbell-lot23** [comment]
March 15 2022 update:

This is a broad and important topic. Whether we allow emojis, colour-coding, custom inline formatting, etc there's definitely a "needle in a haystack" UX question around making the console work better for people.

**Reply by jasonLaster** [reply]
We should differentiate between parity work to support what is in chrome devtools and features we would like to offer on top of that.
